hieradata/production.yaml

```diff
@@ -66,6 +66,9 @@ govuk_bouncer::gor::ip_address: '195.225.216.149'
 govuk_jenkins::config::banner_colour_background: '#ff3300'
 govuk_jenkins::config::banner_colour_text: 'white'
 govuk_jenkins::config::banner_string: 'Carrenza PRODUCTION'
+govuk_jenkins::config::theme_colour: '#ff3300'
```

Comment:
We could use `#df3034` like the publishing apps here:

https://github.com/alphagov/govuk_admin_template/blob/master/app/assets/stylesheets/govuk_admin_template/_theme.scss#L46-L48